test/framework/toolchain.py

```diff
@@ -755,7 +755,7 @@ def test_old_new_iccifort(self):
         self.modtool.purge()
 
         tc = self.get_toolchain('goolfc', version='1.3.12')
-        tc.options.update({'openmp': True})
+        tc.set_options({'openmp': True})
```

Comment:
Yeah, i have no problem with that, i just thought it prudent to make the comment in case you had forgotten :-)

Should i just merge your PR to my fix-multi-flag-compiler-option or was there anything else we needed to do? (loosing track of things due to overload)